diff --git a/glvis-browser-server b/glvis-browser-server
index 44885d7..67c16c1 100755
--- a/glvis-browser-server
+++ b/glvis-browser-server
@@ -93,7 +93,7 @@ async def create_mfem_server(queue):
         await mfem_handler(queue, reader, writer)
 
     server = await asyncio.create_task(
-        asyncio.start_server(handler_wrap, host="localhost", port=args.port)
+        asyncio.start_server(handler_wrap, host="0.0.0.0", port=args.port)
     )
     await server.serve_forever()
     print("mfem server: complete")
